crates/astria-core/src/protocol/transaction/v1alpha1/action.rs

```diff
@@ -33,6 +33,7 @@ pub enum Action {
     Sequence(SequenceAction),
     Transfer(TransferAction),
     ValidatorUpdate(ValidatorUpdate),
+    ValidatorUpdateWithName(ValidatorUpdateWithName),
```

Comment:
Open to calling this action something else if people would like. Maybe something like `NamedValidatorUpdate`